ansible/schedule.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Ensure we have facts about all hypervisors before scheduling begins.
+- name: Gather facts from hypervisors
+  setup:
+  delegate_to: "{{ item }}"
+  delegate_facts: true
+  loop: "{{ groups['hypervisors'] }}"
```

Comment:
nit: I think this can be done using a zero task play, which will ensure it is done in parallel:

```
- hosts: hypervisors
  gather_facts: true
```

Reply:
Okay - would that be best slotted into `deploy.yml` before the play where `schedule.yml` is included?

Reply:
That should work.